packages/@apphosting/adapter-angular/src/interface.ts

```diff
@@ -63,5 +63,4 @@ export const buildManifestSchema = z.object({
     server: z.optional(url),
     browser: url,
   }),
-  prerenderedRoutes: z.optional(z.string().array()),
```

Comment:
Do you think we can comment this out for now and add that the type is different between v19 and v18? Might be helpful later when we actually end up using this field.

Reply:
Done!